### PR TITLE
Updated amp-fill-content to workaround IOS

### PR DESCRIPTION
**Pull request by omegdadi**
This commit addresses the issue described here: https://github.com/ampproject/amphtml/issues/2491

### CODE REVIEW

File: css/amp.css [hunk 1]
```diff
@@ -88,7 +88,8 @@ i-amp-sizer {
 
 .-amp-fill-content {
   display: block;
-  width: 100%;
+  min-width: 100%;
+  width: 1px;
```
Comment (dvoytenko) [inline css/amp.css]: We don't need `width: 1px`, but let's also add `max-width: 100%` - i checked, that works correctly. Please also add a comment and reference https://bugs.webkit.org/show_bug.cgi?id=155198 to explain why we are doing it.

**Comment by dvoytenko** [comment]
Thanks a lot, @omegdadi! I left to requests. PTAL.

**Comment by dvoytenko** [comment]
@omegdadi Thanks! LGTM. Could you please squash the commits and I can then merge.

**Comment by omegdadi** [comment]
NP...squashed.

**Comment by dvoytenko** [comment]
Awesome! Will merge when green. Thanks again!

**Pull request merged**